### PR TITLE
BookA-Pset_BoreholetestPressuremeter

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_BoreholeTestPressuremeter

## Description: 

Properties for results of pressuremeter tests in boreholes.

| Property                    | Datatype                      | Description                                                  |
| --------------------------- | ----------------------------- | ------------------------------------------------------------ |
| MenardPressuremeterModulus  | IfcPressureMeasure            | First loading modulus, resulting from interpretation of Menard's pressuremeter test.|
| MenardLoadingUnloadingRatio | IfcPositiveRatioMeasure       | Reloading modulus, resulting from interpretation of Menard's pressuremeter test.|
| MenardLimitPressure         | IfcPressureMeasure            | Limit pressure, resulting from interpretation of Menard's pressuremeter test.|
| PressiometerType           | IfcLabel                      | Test type (equipment).|
| YoungModulus  | IfcModulusOfElasticityMeasure | According to existing definition |

### CODE REVIEW

**Comment by patmclarin** [comment]
please update description to: Properties for results of pressuremeter tests in boreholes

File: IFC4x3/Properties/e/Emodulus_3n9eJWsrP6tvcwGKZPDBnq/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_3n9eJWsrP6tvcwGKZPDBnq" Name="Emodulus" UniqueId="f12684e0-db56-46df-99ba-4148d934bc74" PrimaryDataType="IfcModulusOfElasticityMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/Emodulus_3n9eJWsrP6tvcwGKZPDBnq/DocProperty.xml]: YoungModulus.

File: IFC4x3/Properties/m/MenardLimitPressure_00Bl8r_Fv7u9veNYw6d5tJ/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Limit pressure, resulting from interpretation of Menard's pressuremeter test.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MenardLimitPressure_00Bl8r_Fv7u9veNYw6d5tJ/Documentation.md]:
```suggestion
The limit pressure, resulting from interpretation of Menard's pressuremeter test.
```

File: IFC4x3/Properties/m/MenardLoadingUnloadingRatio_3NoxGP8K90k8Cg8dlw3vT7/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Reloading modulus, resulting from interpretation of Menard's pressuremeter test.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MenardLoadingUnloadingRatio_3NoxGP8K90k8Cg8dlw3vT7/Documentation.md]:
```suggestion
The reloading modulus, resulting from the interpretation of Menard's pressuremeter test.
```

File: IFC4x3/Properties/m/MenardPressuremeterModulus_3QXrblA1LFxhymk$1IR2rv/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+First loading modulus, resulting from interpretation of Menard's pressuremeter test.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MenardPressuremeterModulus_3QXrblA1LFxhymk$1IR2rv/Documentation.md]:
```suggestion
The first loading modulus, resulting from the interpretation of Menard's pressuremeter test.
```

File: IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/DocProperty.xml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Pressiometer type_1y3JjznIn4hQPFaA5mCFy3" Name="Pressiometer type" UniqueId="7c0d3b7d-c52c-44ad-a64f-90a17030ff03" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/DocProperty.xml]: Quite unconventional naming. Pretty sure it is not allowed even (I could be wrong). Propose renaming to "PressiometerType".

File: IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Test type (equipment).
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/Documentation.md]: Is this description okay? It does not say much to me in terms of explaining the property.

File: IFC4x3/Properties/e/Emodulus_3n9eJWsrP6tvcwGKZPDBnq/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/e/Emodulus_3n9eJWsrP6tvcwGKZPDBnq/DocProperty.xml]: Fixed in [d57a064](https://github.com/bSI-InfraRoom/IFC-Specification/pull/596/commits/d57a064b92b7b5090dacc930a917ae787e7fb48b)

File: IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/DocProperty.xml (hunk 5 above)
Comment (larswik) [reply IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/DocProperty.xml]: Fixed in [d57a064](https://github.com/bSI-InfraRoom/IFC-Specification/pull/596/commits/d57a064b92b7b5090dacc930a917ae787e7fb48b)

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_BoreholeTestPressuremeter/DocPropertySet.xml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_BoreholeTestPressuremeter" UniqueId="fd37b5eb-60a9-4589-86e6-d9d96651eeaa" ApplicableType="IfcGeoScienceObservation/INSITUTESTRESULT" PropertySetType="PSET_OCCURRENCEDRIVEN">
+	<Properties>
+		<DocProperty xsi:nil="true" href="MenardPressuremeterModulus_3QXrblA1LFxhymk1IR2rv" />
+		<DocProperty xsi:nil="true" href="MenardLoadingUnloadingRatio_3NoxGP8K90k8Cg8dlw3vT7" />
+		<DocProperty xsi:nil="true" href="MenardLimitPressure_00Bl8r_Fv7u9veNYw6d5tJ" />
+		<DocProperty xsi:nil="true" href="PressiometerType_1y3JjznIn4hQPFaA5mCFy3" />
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_BoreholeTestPressuremeter/DocPropertySet.xml:7]:
```suggestion
		<DocProperty xsi:nil="true" href="PressuremeterType_1y3JjznIn4hQPFaA5mCFy3" />
```

File: IFC4x3/Properties/p/PressiometerType_1y3JjznIn4hQPFaA5mCFy3/Documentation.md [hunk 8]
```diff
@@ -0,0 +1 @@
+Test type (equipment).
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PressiometerType_1y3JjznIn4hQPFaA5mCFy3/Documentation.md]:
```suggestion
Indication of the type of equipment used for the pressuremeter test.
```

File: IFC4x3/Properties/p/PressuremeterType_1y3JjznIn4hQPFaA5mCFy3/Documentation.md [hunk 9]
```diff
@@ -0,0 +1 @@
+Test type (equipment).
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PressuremeterType_1y3JjznIn4hQPFaA5mCFy3/Documentation.md]:
```suggestion
Indication of the type of equipment used for the pressuremeter test.
```

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**